### ISSUE TITLE
Specify input/output format in Readme

### ISSUE THREAD

**Issue by freenerd** [mapbox/geojson-tidy#1]
The readme should specify exactly which input and output formats exist. Is `coordTimes` a universally accepted property or something we came up with?

**Comment by planemad** [comment]
@freenerd Readme updated, it seems like `coordTimes` is something @tmcw came up with since there is [no standard to do this in the geojson spec](https://github.com/mapbox/leaflet-omnivore/issues/31).

**Comment by tmcw** [comment]
Yep, I would love for this to be slightly more standard but that's a job for mr. @sgillies specification author

**Comment by sgillies** [comment]
Tell me more about `coordTimes`. Sounds like a good candidate for a GeoJSON extension. What it would need to be a good extension is its own namespace, eg, `properties['gpx-ish']['coordTimes']` instead of `properties['coordTimes']`, and documentation of the protocol to be followed (zipping the `coordTimes` and `coordinates` lists together).

**Comment by TheMarex** [comment]
If we are going to to standardize this, we should go with `timestamps` instead of `coordTimes`.

**Comment by planemad** [comment]
Since we're expecting users to use this along with [togeojson](https://github.com/mapbox/togeojson), the change will have to be implemented there first.
